python_modules/libraries/dagster-fivetran/dagster_fivetran/asset_defs.py

```diff
@@ -240,7 +240,9 @@ def _assets(context: OpExecutionContext, fivetran: FivetranResource) -> Any:
     return [_assets]
 
 
-@superseded(additional_warn_text="Use the `fivetran_assets` decorator instead.")
+@deprecated(
+    breaking_version="1.12", additional_warn_text="Use the `fivetran_assets` decorator instead."
```

Comment:
2 versions away from the current version, but it could also be at 2.0. Open to other suggestions